scripts/post-promote-tasks-utils.ts

```diff
@@ -0,0 +1,33 @@
+import fetch from 'node-fetch';
+
+interface IndexableVersions {
+  [channel: string]: string;
+}
+
+interface VersionDiff {
+  channel: string;
+  'amp-version': string;
+}
+
+export async function getVersionDiff(
+  head: string,
+  base: string
+): Promise<VersionDiff[]> {
+  const url = (sha: string) =>
+    `https://raw.githubusercontent.com/ampproject/cdn-configuration/${sha}/configs/versions.json`;
```

Comment:
I wonder what the folks at GitHub would say about us using this raw content URL instead of using the API 🙊
(but who has the time and energy to create an Octokit instance and perform a base64 decode)

Reply:
Yaaa I tried the octokit way first and it was ugly. We use the raw content url in other places too 😇